### DISCUSSION TITLE
2023-07-19 Weekly Meeting 116 Notes/Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#407] [General]
Weekly Meeting 116 is happening on Wednesday, July 19, 2023 from 9:00 AM PST (16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- notes from last week: https://github.com/omigroup/omigroup/discussions/406
- champions two weeks ago: https://github.com/omigroup/omigroup/discussions/404
  - new readme: https://github.com/omigroup/
- MSF POG status 
  - we are approved and finished with the application. 
  - now its time to nominate to the board (I believe, can we confirm?) 
- Open Collective Update 
  - <img width="1510" alt="Screenshot 2023-07-05 at 11 39 24 AM" src="https://github.com/omigroup/omigroup/assets/63426722/8563fc30-b2f3-463f-8d40-59413d180c01">
  - re: payments. I need help. <img width="1126" alt="Screenshot 2023-07-05 at 11 37 47 AM" src="https://github.com/omigroup/omigroup/assets/63426722/0d3a0ab8-0ed9-4bc4-9bc2-9f7ef9c714b9">
- Jin submitted a prop https://github.com/omigroup/omigroup/issues/400, we ready? 
- https://twitter.com/boomboxheads/status/1678872345849794560?s=20 boombox grant? how can we help people like this with exactly this sort of thing? 

### UPDATES:

- vket
- copilot

- MSF delegates
- Media group
- Avatar interop group
- GLTF group
- World Hops

**Comment by mrmetaverse** [comment]
## How can we raise awareness for OMI activities and what is possible? 

Thoughts?

**Reply by mrmetaverse** [reply]
Maybe we can combine forces with other standards bodies and groups. Collectively ride on PR coat-tails of each other.

**Reply by mrmetaverse** [reply]
XR Guild is generally willing to collaborate. 
what larger networks can we tap into as well? 
maybe we present in front of larger audiences? Go where the people are.

**Reply by mrmetaverse** [reply]
Tie virtual and physical events

**Reply by mrmetaverse** [reply]
What if we co-organize a festival or "proper" open metaverse conference of sorts?
Examples: 
- hyperfy.io/neonstreet
- https://www.youtube.com/watch?v=e26UJRCGfGk&pp=ygUlb3BlbiBtZXRhdmVyc2UgaW50ZXJvcGVyYWJpbGl0eSBkZW1vIA%3D%3D
- https://www.youtube.com/watch?v=Hs1jQwXayTk&pp=ygUlb3BlbiBtZXRhdmVyc2UgaW50ZXJvcGVyYWJpbGl0eSBkZW1vIA%3D%3D
- #309 
- ![image](https://github.com/omigroup/omigroup/assets/63426722/91632b1c-d181-4a04-9b70-28233f8503f9)
- https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjfvIP_mJuAAxXWMlkFHUiMBTUQFnoECA0QAQ&url=https%3A%2F%2Fwww.openmetaverseconf.com%2F&usg=AOvVaw2LIkwt46EX-d8r03FQNoHg&opi=89978449
<img width="927" alt="Screenshot 2023-07-19 at 12 33 12 PM" src="https://github.com/omigroup/omigroup/assets/63426722/cdfb644e-2f09-41c8-8f3d-a845bc53e091">
- https://github.com/M3-org/assets

**Reply by mrmetaverse** [reply]
"most of the metaverse conferences happening this year have been in EU & middle east so far; AWE in the US and EU as well"

**Reply by mrmetaverse** [reply]
lets demo how our protocols work (and others like it). Lets integrate a distributed conference with portals.

**Comment by mrmetaverse** [comment]
## Media 
- we have an opp for a AI newsletter tool, if we are interested in using/testing contact Evo!

**Reply by mrmetaverse** [reply]
please record our sessions, create content,and get the word out! https://www.youtube.com/watch?v=KRF6zdmfJHc

**Comment by mrmetaverse** [comment]
## glTF 
- re: microsoft, at an impasse on some critical issues. may result in competing standards. Need to see more people showing up in the comments. At the moment it is mostly 2-3 people discussing and that should be like 10 to be more effective.

**Reply by indiebio** [reply]
Need to email the specific groups who may have an interest in this, wider than "metaverse" interest groups.
 TODO catch up with glTF group to catch up on progress and write something (like an updated table) to send to people.

**Comment by mrmetaverse** [comment]
# We can talk about ourselves 
- how can we dogfood our protocols in our own work? 
- how can we give access to our community to our tools and companies products to experiment further with OMI protocols? 
  - ex. I can give alpha access to magickml.com for anyone who wants to try to experiment with Magick agents and OMI_Personality. 
  - ex. maybe we can encourage builders like null to try to implement OMI protocols in their worlds and VRMs

**Reply by mrmetaverse** [reply]
https://twitter.com/itsmetamike/status/1681293606039207936?s=20

**Comment by mrmetaverse** [comment]
# Conferences are boring - lets have a parade! 
- people can make floats! 
  - maybe the floats use OMI_seat 
- floats can be parked and turned into booths!
- anyone can create a float for the event 
- we can host the parade in multiple platforms 
- a pure celebration of todays interoperability 
- portal jumping across parade floats 😂

**Reply by mrmetaverse** [reply]
this brings together the idea of a modular and reshufflable bazaar

**Comment by mrmetaverse** [comment]
## future standards 
Do we have an OMI_wheel spec? or additional standards for vehicles? 
- yes, we have drafts

**Comment by mrmetaverse** [comment]
# NEXT WEEK 
continue planning next week 
create a page for our website to refer people to about the parade(s)
